torchinductor/lowering.py

```diff
@@ -783,6 +783,8 @@ def inner_fn(index):
 # TODO(jansel): we should implement decomps for these
 # https://github.com/pytorch/torchdynamo/issues/327
 make_fallback(aten._adaptive_avg_pool2d_backward)
+make_fallback(aten.argmax)
```

Comment:
@jansel , is there any specific reason you included argmax here, given that I have added argmax a while ago?

Reply:
No.  I think this was a mistake, feel free to remove it.